### PR TITLE
Added examples in Step Definitions

### PR DESCRIPTION
**Pull request by sajjadhossain**
For behat -di and -dl

### CODE REVIEW

File: src/Behat/MinkExtension/Context/MinkContext.php [hunk 1]
```diff
@@ -392,29 +483,36 @@ public function assertCheckboxChecked($checkbox)
...
+     * Example: Then I should see 5 "div" elements
+     * Example: And I should see 5 "div" elements
      *
-     * @Then /^the "(?P<checkbox>(?:[^"]|\\")*)" checkbox should not be checked$/
-     * @Then /^the checkbox "(?P<checkbox>(?:[^"]|\\")*)" should (?:be unchecked|not be checked)$/
-     * @Then /^the checkbox "(?P<checkbox>(?:[^"]|\\")*)" is (?:unchecked|not checked)$/
+     * @Then /^(?:|I )should see (?P<num>\d+) "(?P<element>[^"]*)" elements?$/
      */
-    public function assertCheckboxNotChecked($checkbox)
+    public function assertNumElements($num, $element)
```
Comment (aik099) [inline src/Behat/MinkExtension/Context/MinkContext.php]: You really want to delete the `assertCheckboxNotChecked` and add new `assertNumElements` in it's place?

**Comment by aik099** [comment]
Can you please post a screenshot of how the `behat -di` and `behat -dl` would look like with and without this PR.

**Comment by sajjadhossain** [comment]
**With Examples**
withexamples

**Without Examples**
withoutexamples

File: src/Behat/MinkExtension/Context/MinkContext.php [hunk 2]
```diff
@@ -44,9 +50,13 @@ public function visit($page)
     }
 
     /**
-     * Reloads current page.
+     * Reloads current page
+     * Example: Given I reload the page
```
Comment (stof) [inline src/Behat/MinkExtension/Context/MinkContext.php]: this is not a Given step. Please don't advocate usage in a Given step. It is semantically wrong (same should be checked for all other steps)

File: src/Behat/MinkExtension/Context/MinkContext.php [hunk 3]
```diff
@@ -44,9 +50,13 @@ public function visit($page)
...
+     * Example: When I reload the page
+     * Example: And I reload the page
      *
      * @When /^(?:|I )reload the page$/
+     *
```
Comment (stof) [inline src/Behat/MinkExtension/Context/MinkContext.php]: please remove this extra line

File: src/Behat/MinkExtension/Context/MinkContext.php [hunk 4]
```diff
@@ -54,7 +64,10 @@ public function reload()
     }
 
     /**
-     * Moves backward one page in history.
+     * Moves backward one page in history
+     * Example: Given I move backward one page
+     * Example: When I move backward one page
+     * Example: And I move backward one page
```
Comment (stof) [inline src/Behat/MinkExtension/Context/MinkContext.php]: A single example would be enough here (Given is wrong, and ``and`` is useless as it is exactly the same)

File: src/Behat/MinkExtension/Context/MinkContext.php (hunk 1 above)
Comment (stof) [inline src/Behat/MinkExtension/Context/MinkContext.php]: Actually, he has not added new methods. He changed the order of existing methods.

@sajjadhossain please revert this change. Methods should not be reordered. It makes the review a pain

**Comment by sajjadhossain** [comment]
Ahh thanks for the candid feedback, I'll address for all instances and keep that in mind moving forward.

File: src/Behat/MinkExtension/Context/MinkContext.php (hunk 1 above)
Comment (sajjadhossain) [inline src/Behat/MinkExtension/Context/MinkContext.php]: revert that specific change mentioned above @stof ?

And can you be a bit more elaborate about "Methods should not be reordered. It makes the review a pain" Should it look like so:

/*
* Description
* Example
* Gherkin Step
*/

**Comment by sajjadhossain** [comment]
@stof lmk if this addresses your initial concerns or if any other issues pop up.

**Comment by everzet** [comment]
Thank you very much for contribution!
